scripts/gen_csv.py

```diff
@@ -0,0 +1,197 @@
+import os
+from xml.etree import ElementTree
+import pandas as pd
+
+class Patient:
+    def __init__(self):
+        
+        self.text = None
+        self.id = 0
+        
+        self.Asthma = None
+        self.CAD = None
+        self.CHF = None
+        self.Depression = None
+        self.Diabetes = None
+        self.Gallstones = None
+        self.GERD = None
+        self.Gout = None
+        self.Hypercholesterolemia = None
+        self.Hypertension = None
+        self.Hypertriglyceridemia = None
+        self.OA = None
+        self.Obesity = None
+        self.OSA = None
+        self.PVD = None
+        self.Venous = None
+
+
+dic = {} 
+
+# read the first training data
+tree = ElementTree.parse('./text1.xml')
+root = tree.getroot()
+docs  = root.findall('docs')[0]
+for doc in docs:
+    text_data = doc.findall('text')[0].text  # extract texxt data
+    pat = Patient() # create pateint
+    pat.text = text_data
+    pat.id = int(doc.attrib['id'])
+    dic[pat.id] = pat
+
+# read the second training data
+tree = ElementTree.parse('./text2.xml')
+root = tree.getroot()
+docs  = root.findall('docs')[0]
+for doc in docs:
+    text_data = doc.findall('text')[0].text  # extract texxt data
+    pat = Patient() # create pateint
+    pat.text = text_data
+    pat.id = int(doc.attrib['id'])
+    dic[pat.id] = pat
+
+# read the annotations 1
+tree = ElementTree.parse('./label1.xml')
+root = tree.getroot()
+diseases  = root.findall('diseases')[0]
+for disease in diseases:
+    dis_name = disease.attrib['name'].split(' ')[0]
+    print(dis_name)
+    for doc in disease:
+        id_ = int(doc.attrib['id'])
+        setattr(dic[id_] , dis_name, doc.attrib['judgment'])
+
+# read the annotations 2
+tree = ElementTree.parse('./label2.xml')
+root = tree.getroot()
+diseases  = root.findall('diseases')[0]
+for disease in diseases:
+    dis_name = disease.attrib['name'].split(' ')[0]
+    for doc in disease:
+        id_ = int(doc.attrib['id'])
+        setattr(dic[id_] , dis_name, doc.attrib['judgment'])
+
+# read the annotations 3
+tree = ElementTree.parse('./label3.xml')
+root = tree.getroot()
+diseases  = root.findall('diseases')[0]
+for disease in diseases:
+    dis_name = disease.attrib['name'].split(' ')[0]
+    for doc in disease:
+        id_ = int(doc.attrib['id'])
+        setattr(dic[id_] , dis_name, doc.attrib['judgment'])
```

Comment:
Same as above, let's make this a loop to avoid repeating the same steps in code.